src/pybind/mgr/mds_autoscaler/module.py

```diff
@@ -25,7 +25,6 @@ def get_service(self, fs_name: str) -> Optional[orchestrator.ServiceDescription]
         completion = self.describe_service(service_type='mds',
                                            service_name=service,
                                            refresh=True)
-        self._orchestrator_wait([completion])
         orchestrator.raise_if_exception(completion)
         if completion.result:
             return completion.result[0]
```

Comment:
```
result = orchestrator.raise_if_exception(completion)
if result:
    return result[0]
```

Reply:
I'd like to keep the changes to the code to a minimum. The PR is already super big.

Reply:
Okay but please consider changing in follow-up PR.